src/main/java/io/kestra/plugin/langchain/OpenAIClassification.java

```diff
@@ -0,0 +1,116 @@
+package io.kestra.plugin.langchain;
+
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import dev.langchain4j.model.openai.OpenAiChatModelName;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.langchain.exceptions.ApiKeyNotFoundException;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "LangChain4j Classification Task",
+    description = "Classifies text using LangChain4j by generating a class label"
+)
+@Plugin(
+    examples = {
+        @io.kestra.core.models.annotations.Example(
+            title = "Classification Example",
+            code = {
+                "prompt: \"Is 'This is a joke' a good joke?\"",
+                "classes: [\"true\", \"false\"]",
+                "model: \"gpt-4\""
+            }
+        )
+    }
+)
+public class OpenAIClassification extends Task implements RunnableTask<OpenAIClassification.Output> {
+
+    @Schema(
+        title = "Text prompt",
+        description = "The input text to classify"
+    )
+    @NotNull
+    private Property<String> prompt;
+
+    @Schema(
+        title = "Classes",
+        description = "The list of possible classes for classification"
+    )
+    @NotNull
+    private Property<List<String>> classes;
+
+    @Schema(
+        title = "Apikey",
+        description = "OpenAI API key"
+    )
+    private Property<String> apikey;
+
+    @Schema(
+        title = "OpenAi model",
+        description = "OpenAi model name"
+    )
+    private Property<OpenAiChatModelName> openAiChatModelName;
+
+    @Override
+    public OpenAIClassification.Output run(RunContext runContext) throws Exception {
+        Logger logger = runContext.logger();
+
+        // Render inputs
+        String renderedPrompt = runContext.render(prompt).as(String.class).orElse("");
```

Comment:
```suggestion
        String renderedPrompt = runContext.render(prompt).as(String.class).orElseThrow();
```
As the property is mandatory, it will always be set.